tests/unit_tests/test_unit.py

```
@@ -567,19 +578,23 @@ def test_setup_model(monkeypatch, model_arg, expected_filename):
         mnk.setattr(requests, "get", mock_get)
 
         with pytest.raises(ValueError):
-            casanovo.setup_model("FooBar", None, None, None, False)
+            shared_loading.setup_model(
+                "FooBar", None, None, None, False, ckpt_regex, version
```

Ruff (0.16.3)

[warning] Boolean positional value in function call

(FBT003)